### DISCUSSION TITLE
1.4.0에서 iOS 딥링크가 동작하지 않는 현상 확인

### DISCUSSION THREAD

**Discussion by psguny9** [kakao/kakao_flutter_sdk#166] [General]
1.4.0 SDK를 사용하면 iOS 앱이 종료된 상태에서 딥링크 실행시 동작하지 않습니다.

SwiftKakaoFlutterSdkPlugin.swift의 

 public func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [AnyHashable : Any] = [:])
 
 에서 return false가 문제가 되고 있습니다. 해당 부분은 항상 true 여야 합니다. 빠른 수정 부탁합니다.

**Comment by mbkim95** [comment]
안녕하세요 확인이 조금 늦었습니다ㅠㅠ

말씀주신 이슈는 [카카오 데브톡 문의](https://devtalk.kakao.com/t/firebase-dynamic-link-kakao-flutter-sdk-ios-dynamic-link/127905), [Github Issue](https://github.com/kakao/kakao_flutter_sdk/issues/169)를 통해 이미 제보된 이슈로 [해당 커밋](https://github.com/kakao/kakao_flutter_sdk/commit/f0e31b45e4e652a62964ce99e04cf9cb6c09c4cf)에서 수정되었습니다.

이슈 수정된 1.4.2 버전은 빠르면 이번 주 말, 늦으면 다음 주 중으로 배포할 예정인 점 참고 부탁드리겠습니다

제보주셔서 감사합니다

**Reply by mbkim95** [reply]
@psguny9 

안녕하세요

제보주신 내용 수정된 1.4.2 버전 배포되었습니다.

이슈 제보주셔서 감사합니다
